cli.js

```diff
@@ -159,3 +204,7 @@ async function safeExecute(func) {
         await func()
     } catch {}
 }
+
+function isNotHiddenFile(src, dest) {
+    return !src.match(/.+\/\..*/)
```

Comment:
Is the idea here to match files like `.gitignore`?

Because if so, it doesn't?

Reply:
It should match every file or folder beginning with a dot. In my testing it is working as intended but I will look into this